Configuration Bitstream Files/QL732B_cdl.cmd

```diff
@@ -0,0 +1,57 @@
+#---------------------------------------------------------------------------------
```

Comment:
Are you adding this file for EOS-S3 information?

Reply:
I noticed that similar file is available in PolarPro3 repository: [QL725A_cdl.cmd](https://github.com/QuickLogic-Corp/PolarPro3/blob/master/Configuration%20Bitstream%20Files/QL725A_cdl.cmd)
We want to use those `_cdl.cmd` files to read device-specific information that will be used in [quicklogic-fasm](https://github.com/QuickLogic-Corp/quicklogic-fasm) for bitstream generation process.

Reply:
please note that part number for EOS-S3 is QL723B but for PP3, part number is QL725A. So, for PP3, we should use QL725A_cdl.cmd file.

Reply:
Yes, that is correct. This file is specifically for EOS-S3 (QL723B).
Formerly the scripts in [quicklogic-fasm](https://github.com/QuickLogic-Corp/quicklogic-fasm) were supporting only EOS-S3 - the scripts had some hardcoded EOS-S3-specific data. This is changed in https://github.com/QuickLogic-Corp/quicklogic-fasm/pull/14 where we add support for FASM db generation and initial bitstream generation for PP3. We get rid of this hardcoded data and replace it with parsing the `*_cdl.cmd` files to get that information.

So adding `QL732B_cdl.cmd` file is not strictly related to PP3 itself but it is needed to parameterize some things in quicklogic-fasm and enable usage of parameterized scripts for EOS-S3 (the usage is [here](https://github.com/QuickLogic-Corp/quicklogic-fasm/pull/14/files#diff-d3ceddb253d55c95ee6c2f9ecb44264bface2ae91e62a822f1da8d0c50ddd2b4R458)).
As I explained before, similar file is placed in PolarPro3 repository so I think that corresponding `*_cdl.cmd` file for EOS-S3 should be available in this repository so that there would be a uniform method for accessing this data for all supported devices.

Reply:
ok